root/static/scripts/common/components/Annotation.js

```diff
@@ -19,17 +19,24 @@ import entityHref from '../utility/entityHref';
 import Collapsible from './Collapsible';
 import EditorLink from './EditorLink';
 
+type EntityProp = $ReadOnly<MinimalCoreEntityT & {
```

Comment:
It’s not really a component property set, it looks more like an entity type, could be named `MinimalAnnotativeEntityT` or something.

Reply:
My intent was for the singular Prop to indicate it's for one prop, not a set. :) But `MinimalAnnotatedEntityT` sounds pretty good (with Annotated instead of Annotative)